ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Please provide more information to help us understand the issue:
+
+* Are you reporting a bug, or opening a feature request?
+* Please insert below the code you are checking with mypy,
+  or a mock-up repro if the source is private. We would appreciate
+  if you try to simplify your case to a minimal repro.
+* What is the actual behavior/output?
+* What is the behavior/output you expect?
+* What are the versions of mypy and Python you are using?
+  Do you see the same issue after installing mypy from Git master?
```

Comment:
Also add instructions for this `python3 -m pip install git+https://github.com/python/mypy`